docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -1295,7 +1295,7 @@ public void validate_shouldMarkClientAsFraud() throws Exception {
 ----
 ====
 
-If you used the Groovy DSL, you can see that all of the `producer()` parts of the Contract that were present in the
+If you used the Groovy DSL, you can see that all the `producer()` parts of the Contract that were present in the
```

Comment:
This one (and the others like it) are a matter of style. Since the change involves fewer words, it's a good change for us. Grammatically speaking, though, both are correct.